### PR TITLE
Remove v1 messages/code [CWC-1651]

### PR DESCRIPTION
**Pull request by tflpd**
## Description of the change

Remove all remaining V1 code/messages to avoid causing confusion to developers implementing new features

## Relevant release note information

Release Notes: Remove v1 messages/code

## Related JIRA tickets

Relates to JIRA: CWC-1651

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [X] No

If yes, please explain:

### CODE REVIEW

**Review by sidpremkumar — APPROVED** [review]
Tested `targetGroup`:
```
zli targetGroup sid-cluster-policy system:master -a
Logged in as: sid@commonwealthcrypto.com, bzero-id:5ba70552-575f-421b-9f0b-93c34cea087f, session-id:090ea2a3-4efd-4300-9e02-b15dbb9f9ef1
Added system:master to sid-cluster-policy policy!
zli targetGroup sid-cluster-policy system:master -d
Logged in as: sid@commonwealthcrypto.com, bzero-id:5ba70552-575f-421b-9f0b-93c34cea087f, session-id:090ea2a3-4efd-4300-9e02-b15dbb9f9ef1
Deleted system:master from sid-cluster-policy policy!
```

Tested user:
```
zli user sid-cluster-policy sid@commonwealthcrypto.com -d
Logged in as: sid@commonwealthcrypto.com, bzero-id:5ba70552-575f-421b-9f0b-93c34cea087f, session-id:090ea2a3-4efd-4300-9e02-b15dbb9f9ef1
Deleted sid@commonwealthcrypto.com from sid-cluster-policy policy!
zli user sid-cluster-policy sid@commonwealthcrypto.com -a
Logged in as: sid@commonwealthcrypto.com, bzero-id:5ba70552-575f-421b-9f0b-93c34cea087f, session-id:090ea2a3-4efd-4300-9e02-b15dbb9f9ef1
Added sid@commonwealthcrypto.com to sid-cluster-policy policy!
```

Tested list user:
```
zli user sid-cluster-policy
Logged in as: sid@commonwealthcrypto.com, bzero-id:5ba70552-575f-421b-9f0b-93c34cea087f, session-id:090ea2a3-4efd-4300-9e02-b15dbb9f9ef1
┌─────────────────────┬──────────────────────────────────────┬───────┬────────────────────┐
│ Name                │ Email                                │ Role  │ Last Login         │
├─────────────────────┼──────────────────────────────────────┼───────┼────────────────────┤
│ Sidhartha Premkumar │ sid@commonwealthcrypto.com           │ Admin │ 4/13/22, 3:26 PM   │
```



Tested list targetGroup
```
zli targetGroup sid-cluster-policy
Logged in as: sid@commonwealthcrypto.com, bzero-id:5ba70552-575f-421b-9f0b-93c34cea087f, session-id:090ea2a3-4efd-4300-9e02-b15dbb9f9ef1
There are no available target groups
```

Tested kube connection:
```
zli connect cluster-admin@sid-cluster
Logged in as: sid@commonwealthcrypto.com, bzero-id:5ba70552-575f-421b-9f0b-93c34cea087f, session-id:090ea2a3-4efd-4300-9e02-b15dbb9f9ef1
Waiting for daemon to become ready...
Started kube daemon at localhost:56456 for cluster-admin@sid-cluster
ctx
error: you did not choose any of the options
kubectl get pods
NAME                                       READY   STATUS             RESTARTS   AGE
bzero-autostart-1649664000-msgmr           0/1     Completed          0          2d2h
bzero-autostart-1649750400-r65mn           0/1     Completed          0          26h
bzero-autostart-1649836800-llg9g           0/1     Completed          0          121m
bzero-autostop-1649635200-bh75t            0/1     Completed          0          2d10h
bzero-chad-05c2bfdf-cfbddf54-2p7nq         3/3     Running            0          121m
bzero-chad-de476e2b-67497567db-bjf7w       3/3     Running            0          121m
bzero-dani-b858c1b8-648dbd69cc-r7258       3/3     Running            0          121m
bzero-dev-7677f77c9f-l4zsq                 2/2     Running            0          18h
bzero-devd-9bc45260-5ccdc7dbc4-24z8n       2/2     Running            1          7d9h
bzero-ethan-a517596e-7c9545ddf9-m5hkt      3/3     Running            0          121m
bzero-john-c6d1c5c7-58b58c655-s4qwd        3/3     Running            0          121m
bzero-lucie-632d41d0-6579665898-j8zqj      3/3     Running            0          121m
bzero-sebby-74eebe2b-566cd479cb-68n6l      3/3     Running            0          121m
bzero-sid-b02b4d95-5cf48fdfdf-g2lvs        3/3     Running            0          121m
bzero-than-6cd7abed-5d5556cb97-wrq4x       3/3     Running            0          121m
bzero-thanasis-56241193-6f7647f9f6-fjshn   3/3     Running            0          121m
bzero-vineet-d4cff77e-5744cfc6c5-9tnqz     3/3     Running            0          121m
bzero-yuval-e3ef1509-74d57c565-69llk       3/3     Running            0          121m
test-deployment-67b6476bb4-kn65s           0/1     CrashLoopBackOff   2369       8d
```

Tested web connection:
2022-04-13 at 3 35 PM

Tested db connection:
```
zli connect test-db
Logged in as: sid@commonwealthcrypto.com, bzero-id:5ba70552-575f-421b-9f0b-93c34cea087f, session-id:090ea2a3-4efd-4300-9e02-b15dbb9f9ef1
Started db daemon at localhost:6100 for test-db
psql -U postgres -p 6100 -h localhost
psql (14.2, server 12.9)
Type "help" for help.

postgres=# \dt
Did not find any relations.
postgres=#
```

**Comment by sidpremkumar** [comment]
Oh also I see you updated the submodule to capture what I think is my portforward fix. I was under the assumption that @johncmerfeld was going to get that in with his one-action-per-kube PR, so just confirming that it should be fine.

File: src/services/policy/policy.services.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        proxyPolicies = await policyHttpService.ListProxyPolicies();
+
+    if (listKubernetesPolicies)
+        kubernetesPolicies = await policyHttpService.ListKubernetesPolicies();
+
+    if (listTargetConnectPolicies)
+        targetConnectPolicies = await policyHttpService.ListTargetConnectPolicies();
+
+
+    return [
```
[the remark this answers is not in the document]
Comment (tflpd) [reply src/services/policy/policy.services.ts:39]: Broken link :/

[the remark this answers is not in the document]
Comment (sidpremkumar) [reply src/services/policy/policy.services.ts:39]: https://github.com/bastionzero/zli/blob/master/src/system-tests/tests/system-test.ts#L173-L176

**Review by sidpremkumar — APPROVED** [review]

**Pull request merged**